Python/other/Moore_Algorithm.py

```diff
@@ -0,0 +1,29 @@
+def counting_fun(array):
```

Comment:
Comment about the problem description here also

Reply:
Okay